vars-workergroup.auto.tfvars.example

```diff
@@ -0,0 +1,45 @@
+# Copyright 2022 Oracle Corporation and/or affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl
+
+# Default parameters
+label_prefix                  = "dev"
+worker_group_size             = 0
+worker_group_mode             = "node-pool"
+worker_group_shape            = "VM.Standard.E4.Flex"
+worker_group_ocpus            = 2  # Ignored for non-Flex shapes
+worker_group_memory           = 16 # Ignored for non-Flex shapes
+worker_group_boot_volume_size = 100
+
+# Default parameters when undefined for a group
+worker_group_enabled           = false # Defaults to true; must otherwise enable groups explicitly
+worker_compartment_id          = "" # Defaults to compartment_id when empty
+worker_group_primary_subnet_id = "" # Defaults to Terraform-managed when empty
+
+# Some default worker_groups vars will fall back to node_pools values when != "none"
+node_pool_image_id   = "ocid1.image..." # Ignored when image type != "custom"
+node_pool_image_type = "oke" # Must be "custom" when providing an image OCID
+node_pools  = {} # Functions as before; only worker_groups are defined in this example
```

Comment:
for complex types, include all variations of node pools

Reply:
Updated examples; more to be added when associated support is available.